### ISSUE TITLE
Team Management with common team members

### ISSUE THREAD

**Issue by BhadreshDoshi** [adarshmadrecha/jamku#162]
Currently there is no option to form a particular team and give a title to it. It might be possible that one team member is tagged under two different team leaders (in a small organisation) and he is executing tasks assigned by both the TLs. So, the tasks assigned to that team member will get reflected in the dashboard of both the TLs. This can be avoided by adding a feature of team creation and allowing dashboard to show the tasks on the basis of teams which are selected.

**Comment by vatsalbhatia** [comment]
This would be of great help

**Comment by G-Kumaran** [comment]
Agreed, as most mid size offices have fluid team structure and not fixed teams

**Comment by adarshmadrecha** [comment]
We had a log conversation in design meeting on need of this feature.
The problem sugges by @BhadreshDoshi can easily be resolved by using "CostCenter" feature of Jamku.
